### PR TITLE
Change to custom color scheme when overriding a color

### LINKED ISSUE #42: Show color scheme as Custom when overridden
When a user selects a Color Scheme and then decides to override colors in it, the Scheme name should dynamically change to "- Custom -" to give a visual indication that they have diverged from the actual scheme they originally selected.

screen shot 2016-07-28 at 4 27 03 pm

### PR DESCRIPTION
**Pull request by jonathanbardo**
This should fix #42 

@fjarrett Please test.

### CODE REVIEW

**Comment by jonathanbardo** [comment]
@fjarrett Are you happy with this?

**Comment by frankiejarrett** [comment]
@jonathanbardo I would really prefer it if the `- Custom -` option was not listed in the dropdown unless the user has diverged from an actual color scheme.

**Comment by jonathanbardo** [comment]
@fjarrett You are right, please test latest code push.

**Comment by frankiejarrett** [comment]
@jonathanbardo There are a couple issues:
1. The `- Custom -` item is added every time you select a new color: https://youtu.be/9YgHMD8zLlU
2. If you change a color, then click "Default" to go back, the dropdown should change back to the Color Scheme you had originally selected and `- Custom -` should disappear.

**Comment by jonathanbardo** [comment]
@fjarrett I've fix the issues you found. Please test again.

**Comment by frankiejarrett** [comment]
@jonathanbardo Looking very nice. Only last I bug I see is that the `- Custom -` selection does not persist in future Customizer sessions.

https://youtu.be/BjlVlsDIv7w

**Comment by jonathanbardo** [comment]
@fjarrett Alright, custom is now persistent across customizer session. I have a feeling we should let the user go back to their previous custom choice after they create one. This might be another enhancement though.

File: inc/customizer/colors.php [hunk 1]
```diff
@@ -466,9 +474,11 @@ public function color_scheme_control_js() {
 
 		$suffix = SCRIPT_DEBUG ? '' : '.min';
 
-		wp_enqueue_script( 'primer-color-scheme-control', get_template_directory_uri() . "/assets/js/color-scheme-control{$suffix}.js", array( 'customize-controls', 'iris', 'underscore', 'wp-util' ), PRIMER_VERSION, true );
+		wp_enqueue_script( 'primer-color-scheme-control', get_template_directory_uri() . "/assets/js/admin/color-scheme-control{$suffix}.js", array( 'customize-controls', 'iris', 'underscore', 'wp-util' ), PRIMER_VERSION, true );
+
+		$color_schemes = $this->color_schemes;
 
-		wp_localize_script( 'primer-color-scheme-control', 'colorSchemes', $this->color_schemes );
+		wp_localize_script( 'primer-color-scheme-control', 'colorSchemes', $color_schemes );
```
Comment (frankiejarrett) [inline inc/customizer/colors.php]: @jonathanbardo I don't really get the change here. Is there a reason?

Comment (jonathanbardo) [inline inc/customizer/colors.php]: My bad! I added some code and then I removed it later.

**Comment by frankiejarrett** [comment]
@jonathanbardo You are awesome. This works great.

**Pull request merged**